apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -114,14 +114,10 @@ export default function getRowsKeyboardNavigation({
         break;
       case KEYS.ENTER:
         confirm();
-        break;
+        return;
       case KEYS.ESCAPE:
-        if (isModal()) {
-          cancel();
-        } else {
-          return; // propagate to other Esc handler
-        }
-        break;
+        cancel();
+        return;
```

Comment:
I think this will not work for keyboard navigation of inline listboxes in filterpane.
then you probably not want to propagate escape when selection is aborted

Reply:
This change is not actually part of the bug-report but I found that press Esc when you have some selections crashes the listbox, it is an obvious error. I think Esc should close the listbox under any circumstances(with/without selections), it is how it works on the old table. I just tried to keep the compatibility, unless it is not how it is designed to work with the nebula listbox?

Reply:
ok, then it makes sense to not to change Esc like this. 
I'll remove Esc related changes since it was not part of the bug report and will take care of it in another PR.